framework/resources/Functional/filter/selectivity/plan/q1.res

```diff
@@ -0,0 +1,3 @@
+.*Screen.*
```

Comment:
This looks like a regular explain plan verification right? It should be outside of the filter selectivity tests.